### DISCUSSION TITLE
Can I download the tailwind.css file and use?

### DISCUSSION THREAD

**Discussion by FredrikGehrke** [tailwindlabs/tailwindcss#9737] [Help]
Hi, haven't used Tailwind in a couple months and I want to remember that I somehow could download the whole taildwind.css file and use instead of installing via CLI or CDN. I don't get it working with CLI.. think I'm doing the step 4 wrong.

Is there a while .css file I can download and put into my project instead?
I'm only using a simple index.html file and want to use tailwinds styling without using CDN.

**Comment by wongjn** [comment]
Not for v3 – there's an infinite amount of classes possible (due to the arbitrary features) and even without these, if you really wanted every single class possible, you would need to generate every possible combination of the variants, (like `dark:first:hover:focus:md:visited:last:only:…:m-1`) so there would be all the "base" classes plus all these permutations of variants and thus the total number of possible classes would be something like (70![^1] + 1) × number of "base" classes (maths probably not correct here, but something like this). The "real" number would be slightly less since some variants don't make sense together, i.e. `sm:md:lg:xl:m-1` but this obscenely large number should demonstrate the point.

[^1]: 70 factorial, equal to 11978571669969891796072783721689098736458938142546425857555362864628009582789845319680000000000000000
